netq.h

```diff
@@ -39,6 +39,10 @@
 #endif
 #endif
 
+typedef enum netq_job_type_t {
```

Comment:
The new `netq_job_type` and the new field `job` in `netq_t` should be documented.

Reply:
OK. I will do it a little later in the afternoon.